force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.js

```diff
@@ -0,0 +1,112 @@
+import { LightningElement, api, track } from "lwc";
+import accountModelSettingsTitle from "@salesforce/label/c.stgAccountModelSettingsTitle";
+
+export default class EdaSettingsNavigation extends LightningElement {
+    labelReference = {
+        accountModelSettingsTitle,
+    };
+
+    @track settingsNavigationViewModel = {
+        navigationSections: [
+            {
+                label: "Setup Home",
+                page: "setuphome",
+                sectionkey: "setuphome",
+            },
+            {
```

Comment:
Do we need to translate the text for the navigation section labels?

Reply:
We will eventually. These are placeholders to be replaced as we go and add the actual pages. The page and sectionkey can remain untranslated as they are back-end